pv_site_api/main.py

```diff
@@ -77,11 +82,11 @@ def is_fake():
     return int(os.environ.get("FAKE", 0))
 
 
-sentry_sdk.init(
-    dsn=os.getenv("SENTRY_DSN"),
-    environment=os.getenv("ENVIRONMENT", "local"),
-    traces_sampler=traces_sampler,
-)
+# sentry_sdk.init(
```

Comment:
why remove this?

Reply:
Definitely did not mean to leave this in, thanks for catching